tools/mo/unit_tests/mo/utils/cli_parser_test.py

```diff
@@ -469,6 +469,22 @@ def test_get_shapes_several_inputs_several_shapes2(self):
         for i in placeholder_values_ref.keys():
             npt.assert_array_equal(placeholder_values_res[i], placeholder_values_ref[i])
 
```

Comment:
please also add a check for the case without type: node_name->False.
And in this case please check data type also

Reply:
Partially replied in other comment. Here, if user didn't specified data type we can not check it. And we also here do not have access to the original model data types. Such things will be handles in 
https://github.com/openvinotoolkit/openvino/blob/master/tools/mo/openvino/tools/mo/front/freeze_placeholder_value.py#L48-L57

`node_name->False` - we already have such cases in unit-tests. 
https://github.com/openvinotoolkit/openvino/blob/master/tools/mo/unit_tests/mo/utils/cli_parser_test.py#L858-L859

But there was a bug when such freezing was done without specifying any shape in `--input`. So I added only one additional test case for the scenario when there are no shapes in `--input`

Reply:
I think we need to align it with `freeze_placeholder_with_value` option through which we can specify boolean value for placeholders and its type determined correctly. 
I propose to take a model that requires this case (for example, in order to avoid conditional flows) and use --input option instead of `freeze_placeholder_with_value`